shared/js/ui/templates/shared/hamburger-button.es6.js

```diff
@@ -2,7 +2,7 @@ const bel = require('bel')
 
 module.exports = function (klass) {
   klass = klass || ''
-  return bel`<button class="hamburger-button ${klass}">
+  return bel`<button type="button" class="hamburger-button ${klass}">
```

Comment:
cc @jdorweiler this and https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/195/files#diff-6a64f88be34c8f8036a1faf3c3192e6bR35 fix the hamburger menu problem.

Reply:
👍 thanks